CommandHelper/LocalPackages/commands/buffs/toppo.ms

```diff
@@ -0,0 +1,37 @@
+@plr = player()
+
+
+@hero = associative_array(
+            'name': '§4T§co§epp§co',
+            'class': 'toppo',
+            'sig': 'eyJ0aW1lc3RhbXAiOjE1MTgyNzU0ODI1MDIsInByb2ZpbGVJZCI6IjA4ODBlY2NjZmVlZDRhMmY5ZTIyZDhmNzIxZjM5MTNlIiwicHJvZmlsZU5hbWUiOiJaZUVuZm9yY2VyIiwic2lnbmF0dXJlUmVxdWlyZWQiOnRydWUsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS83ZDU4ZmNhYzM5YjY4MzYwNTkzMTNjMTMzYTg0MjIxZmMyYmExN2VhMzZmZjM3ZjdlZTMzOWZjMDUyZDU0NyJ9fX0='
+            'val': 'rCbYbUC0L7jPINP8S6CNfXD79bAR/iZ5uLQLSWJM0mfN5mqoWm7Z57ZpOa0p6ekS2BiQ3pn08sT5yk7soExfZz05vtFAywgW5se6cP+wo4lQ6cRiLdvCSFNmYsnJkfcJjPzwlOnLR8euSnB9UzcS+Br00YHhLMoOwE4RuzfShEbL6xG89XEcE/7AMboU820PvaKAFDUdV6ymjYlcpG/FiwVw+avPNw15svesqkqniab6zphZvXKcQ9YvcqoNvnV3vVhsegGpcALpLeCQafp23IfLv3t2bQHE8EpkBfkv/4cubNCjEC9SYqQeDLN7x/5LpNw4A8m1xYuNh/9YrYnLxUwmZVBaGw+rqLEdYyAvwiDatTKSd5SzT2vnvMEv7eF5aclvN6sDWv8/28xtueWcuCqPZyiirxDNbgVIb8ho2iaYUvFTtn0wAwmuK55KMcXuxVtIyK+tUDtGcyqKS6+bPr7ITcQqbJn+ArGoL+IPLS6qA7mkB0oZvGOEQdp9sTv6smWR6XsZVNeccMFj62qdTsN742msdPv9QQ29BhOPtZoyFaXVRClhKH/6vob7vd5Q5/NzRNtyGBVWZlbTwgExtfcfx2EnWAZThLCczCNCVGG8gruluCDEQazoewtNKoBYmpQu7+CeUdshLe0rNjhSZkO5Gij+ri34SfgY+WZdE8I='
+            'trail': '',
+            'op': 'false',
+            'speech': "You cannot beat a destrution GOD!!",
+            'powers': array(
+                "/buff",
+                "/jump 0-9",
+                "/run 0-9",
+                "/haste 0-9",
+                "/flyspeed <number>",
+                "/fly",
+                "/carryplayer <player>",
+                "/dismountplayer"
+                "/destroyworld"
+                "Anything you attack, dies"
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 30 true)",
+                "/effect ". @plr ." 10 9999 1 true",
+                "/effect ". @plr ." 11 9999 10 true",
+                "/effect ". @plr ." 12 9999 1 true",
+                "/effect ". @plr ." 13 9999 1 true",
+                "/effect ". @plr ." 21 9999 19 true",
+                "/shot give ". @plr ." destructionbeam",
+                "/shot give ". @plr ." kiblast"
+            ),
+            'alts': array(
+                'toppo'
+            )
```

Comment:
why did you add a transformation to transform him into toppo? he's already toppo, no transformation needed